test/bandit/http1/request_test.exs

```diff
@@ -2150,7 +2183,7 @@ defmodule HTTP1RequestTest do
 
       assert output =~ "(Bandit.HTTPError) closed"
       refute output =~ "IMPOSSIBLE"
-      assert ThousandIsland.connection_pids(context.server_pid) == {:ok, []}
+      assert ThousandIsland.connection_pids(context[:server_pid]) == {:ok, []}
```

Comment:
I had to add this because I believe the `http_server` converts context from a map into a keyword list. I was getting the following error, so I had to make this change:
```
** (KeyError) key :server_pid not found in: [base: "http://localhost:35985", port: 35985, server_pid: #PID<0.531.0>]

     If you are using the dot syntax, such as map.field, make sure the left-hand side of the dot is a map
```